client/src/pages/profile/Profile.tsx

```diff
@@ -34,6 +34,7 @@ function Profile(): ReactElement {
   const [edit, setEdit] = useState<boolean>(false);
   const [error, setError] = useState<boolean>(false);
   const [errorMessage, setErrorMessage] = useState<string>('');
+  const userId = '6031a866c70ec705736a79e5';
```

Comment:
This user ID seems to be getting delcared in a lot of different places around this project, the current User endpoint exists already, is there a reason we aren't using that?

Reply:
I think Aditya has a separate PR in the works for this, I told him to focus on claiming pitches for now